src/databricks/labs/blueprint/paths.py

```diff
@@ -18,6 +18,7 @@
 from typing import BinaryIO, Literal, NoReturn, TextIO, TypeVar
 from urllib.parse import quote_from_bytes as urlquote_from_bytes
 
+import chardet
```

Comment:
This import means it's not an optional dependency, which is why the downstream projects are failing.